src/en/charms-fan.md

```diff
@@ -0,0 +1,202 @@
+Title: Juju and FAN networking
+TODO:  bug tracking: https://bugs.launchpad.net/juju/+bug/1733354 (remove constraints Note if no longer required)
+       hardcoded: default AWS instance type of 'm3.medium'
+       consider a third example: manually configure the FAN with Azure
+
+# Juju and FAN networking
+
+FAN networking addresses a need raised by the proliferation of container usage
+in an IPv4 context: the ability to manage the address space such that network
+connectivity among containers running on separate hosts is achieved.
+
+Juju integrates with the FAN to provide network connectivity between containers
+that was hitherto not possible. The typical use case is the seamless
+interaction between deployed applications running within LXD containers on
+separate Juju machines.
+
+## FAN overview
+
+The FAN is a mapping between a smaller IPv4 address space (e.g. a /16 network)
+and a larger one (e.g. a /8 network) where **subnets** from the larger one (the
+*underlay* network) are assigned to **addresses** on the smaller one (the
+*overlay* network). Connectivity between containers on the larger network is
```

Comment:
Underlay is the smaller (/16) one, overlay is the larger one (/8). I always mess them up, https://wiki.ubuntu.com/FanNetworking has a nice image to explain.

Reply:
Me too. Thanks.